src/sales/order-archive.md

```diff
@@ -52,6 +52,9 @@ _[Move Order to Archive]({% link configuration/sales/sales.md %})_
     ![]({% link images/images-ee/order-select-in-grid.png %}){: .zoom}
     _Mark Checkbox_
 
+{:.bs-callout-tip}
+Only orders with **Complete** or **Closed** status can be moved to the archive. Orders with other statuses will remain in the Order Management grid.
```

Comment:
Technically an order with any other status can be moved to the archive. I would say it rather depends on the `Order Statuses to be Archived` system config.

Reply:
Hi @dmytro-ch, thank you for the great tip.

I have modified this tip, according to your recommendation, could you please check and approve?

Reply:
Hello @EvheniiaKolkhiieva,

Awesome, thank you!